sharding-core/sharding-core-common/src/main/java/org/apache/shardingsphere/core/constant/properties/ShardingProperties.java

```diff
@@ -62,8 +62,6 @@ private void validate() {
                 errorMessages.add(getErrorMessage(shardingPropertiesConstant, value));
             } else if (type == int.class && !StringUtil.isIntValue(value)) {
                 errorMessages.add(getErrorMessage(shardingPropertiesConstant, value));
-            } else if (type == long.class && !StringUtil.isLongValue(value)) {
```

Comment:
Why remove this, is there any relationship with this issue?

Reply:
sorry, I changed this before, I have modified